projects/Amlogic/bootloader/install

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
```

Comment:
As this file is sourced:

https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/tools/u-boot/package.mk#L60

I'd remove the `#!/bin/bash` - it gives the impression this is a standalone script when in fact it will never work as such.